### ISSUE TITLE
Migrate bower dependencies to npm

### ISSUE THREAD

**Issue by anshumanv** [fossasia/susi_desktop#47]
**Actual Behaviour**

Uses bower.json as an additional package manager

**Expected Behaviour**

just using npm dependecies should suffice, why use 2 package managers

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
